numpy/core/tests/test_multiarray.py

```diff
@@ -1185,6 +1185,21 @@ def test_object_initialized_to_None(self, function, dtype):
         expected = expected * (arr.nbytes // len(expected))
         assert arr.tobytes() == expected
 
+    @pytest.mark.parametrize("func, func_input",
+            [(np.array, [1, 2, 3]),
+             (np.asarray, [1, 2, 3]),
+             (np.asanyarray, [1, 2, 3]),
+             (np.ascontiguousarray, [1, 2, 3]),
+             (np.asfortranarray, [1, 2, 3]),
+             (np.zeros, 5),
+             (np.ones, 5),
+             (np.arange, 5),
+             (np.frombuffer, np.zeros(5),),
+             ])
+    def test_like_as_none(self, func, func_input):
+        arr = func(func_input, like=None)
+        expected = func(func_input)
+        assert_array_equal(arr, expected)
```

Comment:
The test looks good, but can you move it to `test_overrides`.  That also has a working parametrization that covers all functions with `like=` that we can reuse.